tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Install Apache
-  apt: pkg=apache2={{ apache_version }} state=present
+  apt: pkg="apache2={{ apache_version }}" state=present
```

Comment:
From what I've noticed, the quoting is required _because_ a variable is being included.